photowall/app/scripts/app.js

```diff
@@ -119,15 +119,19 @@
         // paper-dialog and not from one of its children.
         if (e.target == dialog) {
           dialog.querySelector('[autofocus]').focus();
-          app.$.main.setAttribute('aria-hidden', true);
-          app.$.main.querySelectorAll(photowall.TABINDEX_SELECTOR)
-              .forEach(function(el) {
-            el.tabIndex = -1;
-          });
+          if(app){
```

Comment:
app must be defined because you're calling send on it?
If so just: `if (app.$.main)`